packages/ember-routing/lib/services/router.js

```diff
@@ -6,6 +6,23 @@ import { shallowEqual, resemblesURL, extractRouteArgs } from '../utils';
    The Router service is the public API that provides component/view layer
    access to the router.
 
+   A simple example of usage:
+
+   ```javascript
+   import Component from '@ember/component';
+   import { inject } from '@ember/service';
+
+   export default Component.extend({
+     router: inject(),
+
+     actions: {
+       next() {
+         this.get('router').transitionTo('other.route');
```

Comment:
also I think the community is moving towards using the get function, so 
```javascript
import { get } from '@ember/object';
...
get(this, 'router').transitionTo('other.route');
```

Reply:
I think we still use `this.get` in the docs https://emberjs.com/api/ember/3.3/functions/@ember%2Fservice/inject